### PR TITLE
Update global-ci allowing set operator branch

### PR DESCRIPTION
**Pull request by aufi**
Update of global-ci action to allow set operator branch name used for install/setup scripts.

### CODE REVIEW

File: .github/workflows/global-ci-bundle.yml [hunk 1]
```diff
@@ -14,6 +14,13 @@ on:
         type: string
         required: false
         default: ""
+      operator_branch:
```
Comment (aufi) [inline .github/workflows/global-ci-bundle.yml:17]: Not sure if this needs to be adjusted also in `bundle`, but added as optional/with default value should not  break it.

**Review by djzager — COMMENTED** [review]
Looking at https://github.com/orgs/community/discussions/25246 I don't know that this is going to work as expected. Have you tested this?

We may have to version control the global-ci workflows to accomodate this use case.

**Review by djzager — APPROVED** [review]

**Comment by aufi** [comment]
Currently getting a failure (for release-0.3)
```
 - ansibleResult:
      changed: 0
      completion: 2024-06-11T13:43:26.90194
      failures: 1
      ok: 15
      skipped: 56
    lastTransitionTime: "2024-06-11T13:43:27Z"
    message: 'Failed to create object: b''{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"Addon.tackle.konveyor.io
      \\"analyzer\\" is invalid: spec.image: Required value","reason":"Invalid","details":{"name":"analyzer","group":"tackle.konveyor.io","kind":"Addon","causes":[{"reason":"FieldValueRequired","message":"Required
      value","field":"spec.image"}]},"code":422}\n'''
    reason: Failed
    status: "True"
    type: Failure
```

**Comment by aufi** [comment]
I was able reproduce this issue locally when trying install release-0.4 Konveyor directly from operator, seems to be an issue on operator side.

**Comment by aufi** [comment]
More notes: https://gist.github.com/aufi/9cdc9bfff2f823ffc01fc00a76eca3e5

**Comment by aufi** [comment]
Looks to work for me locally, merging.

**Pull request merged**